### ISSUE TITLE
Documentation: Usage in (plain) web 

### ISSUE THREAD

**Issue by Vlaaaaaaad** [aws-samples/amazon-cognito-passwordless-auth#1]
Hi,

Awesome sample code y'all built here ❣️ Thank you!

As somebody that's considering using Cognito in a plain/vanilla-JavaScript project, I'd really appreciate [the still-to-be-written docs](https://github.com/aws-samples/amazon-cognito-passwordless-auth/blob/156f9233cbee9ff1b3519e9b1308da08b2119ab0/client/README.md?plain=1#L3). I am creating this issue to show/track demand 🙂 

https://github.com/aws-samples/amazon-cognito-passwordless-auth/blob/156f9233cbee9ff1b3519e9b1308da08b2119ab0/client/README.md?plain=1#L3

**Comment by HasseJohansen** [comment]
Hi

Thank you for building this

I am also trying to use this from plain javascript(I am not experienced writting javascript and I don't know react)

I do get a successful login with a magic link following your example (after changing username to usernameOrAlias)

But how do I track the logged in state? it doesn't seem like signInWithLink() (and by that signedIn) is tracking that and as I can see a comment in the React code telling that signInWithLink without a magic is a no-op will it ever then reach catch in this example?

In the React code it seems like you pass a statusCb and tokensCb. Can I do something similar?

**Comment by ottokruse** [comment]
You're right you need to use the `tokensCb` for that. I updated my example to show. Let me know if that helps.

**Comment by menkari** [comment]
Has anyone here taken a stab at porting `hooks` to an appropriate angular service? I'm looking to port this across with some of the niceties of the scheduling token refresh etc
